overlord/snapstate/snapmgr.go

```diff
@@ -450,15 +450,23 @@ func autoRefreshInFlight(st *state.State) bool {
 	return false
 }
 
-func lastRefresh(st *state.State) (time.Time, error) {
+func (m *SnapManager) LastRefresh() (time.Time, error) {
```

Comment:
they should grow a doc comment I think, that also explains they need to be used with the state lock